infra/singletons/tofu.tf

```diff
@@ -0,0 +1,38 @@
+terraform {
+  required_providers {
+
+    sops = {
+      source  = "carlpett/sops"
+      version = "~> 0.5"
+    }
+
+    github = {
+      source  = "integrations/github"
+      version = "~> 6.0"
+    }
+
+    cloudflare = {
+      source  = "cloudflare/cloudflare"
+      version = "~> 4.0"
+    }
+  }
+  /*
+  backend "s3" {
+    bucket         = "gpo-terraform-state"
+    key            = "infra/singletons/terraform.tfstate"
+    region         = "ca-central-1"
+    dynamodb_table = "terraform-state-locks"
+    encrypt        = true
+    profile        = "gpo"
+  }*/
```

Comment:
Is this intentional? It looks like the other `tofu.tf` files have backends set up

Reply:
ah yeah I was testing something, will revert